### ISSUE TITLE
Erratic result with testIpv4NotLocalNetwork

### ISSUE THREAD

**Issue by remicollet** [fzaninotto/Faker#1146]
This test sometime fails.

```
     public function ipv4()
    {
        return long2ip(mt_rand(0, 1) == 0 ? mt_rand(-2147483648, -2) : mt_rand(16777216, 2147483647));
    }
```

Which means value from 128.0.0.0 to 255.255.255.254 or from 1.0.0.0 to 127.255.255.255

```
    public function testIpv4NotLocalNetwork()
    {
        $this->assertNotRegExp('/\A1\./', $this->faker->ipv4());
    }

So, when ipv4 start with 1., test fails.
```

Looks like there is a bad logic there.

**Comment by remicollet** [comment]
If ipv4 should not return "local" adresses (like the one generated by localIpv4), the ipv4 code should be fixed, else the test is not revelant.

**Comment by fzaninotto** [comment]
Fixed by #1269
